File: ejercicios/AMarotoChivite/AMarotoChivite-01-10-Catan-Java/src/test/java/gameCatan/factories/HumanFactoryTest.java
package gameCatan.factories;

import gameCatan.models.Human;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertNotNull;

class HumanFactoryTest {

    @Test
    void create() {
        Human humanTest = HumanFactory.create();
        assertNotNull(humanTest);
    }
}

File: ejercicios/AMarotoChivite/AMarotoChivite-01-10-Catan-Kotlin/src/test/kotlin/gameCatan/factories/HumanFactoryTest.kt
package gameCatan.factories

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

internal class HumanFactoryTest {
    @Test
    fun create() {
        val humanTest = HumanFactory.create()
        Assertions.assertNotNull(humanTest)
    }
}